### PR TITLE
add append example

### PR DESCRIPTION
**Pull request by TheOnAndOnlyZenomat**
Because of my previous confusion in #440, I have written an example to help other avoid that confusion and to make it easier to find this functionality.

### CODE REVIEW

File: examples/append.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    for file in files {
+        append_zip
+            .start_file(file.to_string_lossy(), Default::default())
+            .unwrap();
+
+        let mut f = File::open(file).unwrap();
+        f.read_to_end(&mut buf).unwrap();
+
+        append_zip.write_all(&buf).unwrap();
+        buf.clear();
```
Comment (a1phyr) [inline examples/append.rs:58]: I think that it would be better to use `std::io::copy` here

Comment (TheOnAndOnlyZenomat) [reply examples/append.rs]: I pushed a commit. I have a suspicion, but could you elaborate as to why `io::copy` is better suited?

Comment (TheOnAndOnlyZenomat) [reply examples/append.rs]: Yikes, yeah, that makes indeed sense

**Comment by Pr0methean** [comment]
Replaced with https://github.com/zip-rs/zip2/pull/55.

**Pull request closed without merging**